test/test_core.py

```diff
@@ -607,3 +792,39 @@ def test_singleton(self):
         self.assertEqual(InitialNamingContext(), InitialNamingContext())
         initial_naming_context.bind('test', object())
         self.assertEqual(initial_naming_context._bindings, InitialNamingContext()._bindings)
+
+    def test_resolve(self):
+        super().test_resolve()
+
+        # Verify that the constant naming contexts are available by default on the initial context:
+        # * Boolean values
+        self.assertEqual(self.context.resolve(('constants', 'True')), True)
```

Comment:
ik zou geneigd zijn van hier de 'javascript achtige' string representatie te kiezen : true, false, null ...

Reply:
Fixed in 2bb75d6cf21e77b538b6bc34c518a5e58e2fca72.